### DISCUSSION TITLE
Import from notebook?

### DISCUSSION THREAD

**Discussion by ryanmckaytx** [observablehq/framework#1600] [Q&A] [answered]
I'm not sure I'm using the right terminology.  I would like to import some of the chart types from https://observablehq.com/@hamzaamjad/tiny-charts.  Is this possible?
I tried
```
import {line} from @hamzaamjad/tiny-charts
import {line} from "observablehq.com/@hamzaamjad/tiny-charts"
import {line} from "https://observablehq.com/@hamzaamjad/tiny-charts"
import {line} from npm:@hamzaamjad/tiny-charts
```
and all got some variation of 
`RuntimeError: Failed to resolve module specifier '@hamzaamjad/tiny-charts'`

**Comment by mbostock** [comment, marked as answer]
Importing from notebooks isn’t supported; you can only import vanilla JavaScript modules. We recommend that you instead convert the code to vanilla JavaScript. See the [Converting notebooks](https://observablehq.com/framework/convert) guide.

**Reply by ryanmckaytx** [reply]
Thank you!
